src/helpers.rs

```diff
@@ -1216,6 +1216,19 @@ where
     throw_ub_format!("incorrect number of arguments: got {}, expected {}", args.len(), N)
 }
 
+/// Check that the number of args is at least the minumim what we expect
+pub fn check_min_arg_count<'a, 'tcx, const N: usize>(
+    name: &'a str,
+    args: &[OpTy<'tcx>],
+) -> InterpResult<'tcx, &'a [OpTy<'tcx>; N]> {
+    if args.len() >= N {
+        let ptr = args.as_ptr() as *const [OpTy<'tcx>; N];
+        // SAFETY: ok because we just checked that the length fits
+        return interp_ok(unsafe { &*ptr });
```

Comment:
Please do this without unsafe code:
```rust
return interp_ok(args[..N].try_into().unwrap());
```

Reply:
There is a better option, `split_first_chunk`.

Reply:
Ah, that's very nice. :)